### DISCUSSION TITLE
Comparison with other OpenAPI3 generators (namely ogen as of now)

### DISCUSSION THREAD

**Discussion by Liooo** [oapi-codegen/oapi-codegen#1027] [General]
Initiating the classic "how does this compare to ~" thread to discuss the OpenAPI3 compliant server code generators. 
As far as I know, oapi-codegen and ogen (https://github.com/ogen-go/ogen) are the sole two "production ready" options out there, and given oapi-codegen has got more users and Github stars, I'm curious about why people chose oapi-codegen over ogen (or just weren't aware of it). On the flip side, if anyone here has opted for ogen, I'd love to hear your thoughts and reasons for that choice. Thanks.

**Comment by develmac** [comment]
Not sure if this project is still alive tbh, just looking at the 97 open PRs. Ogen seems to be alive and kicking.

**Reply by jamietanna** [reply]
This project is still alive, just being maintained in two very busy folks' free time. As we've got wider use than ogen we've got a lot more to work through and improve - we're trying our best ☺

I've been meaning to privately chat with @deepmap-marcinr about seeing how we can keep on top of some of the requests, and as ever, having folks in the community helping triage PRs and Issues is always very recommended! That's how I got involved in maintaining, and at a minimum, helps other users solve their issues quicker!

**Reply by johnshiver** [reply]
might help if this project added a contributing guide. I use this project at work / would be happy to help more but feel a bit lost as to where to start

**Comment by sonu27** [comment]
This project is still alive. But I think we need more maintainers. I definitely like the aspect of no pointers with ogen. I will try it out
